assets/index.json

```diff
@@ -1,9 +1,14 @@
 {
   "plugins": [
     {
-      "name": "Cadastrapp",
+      "name": "Cadastrapp1",
```

Comment:
```suggestion
      "name": "Cadastrapp",
```

Reply:
I was testing by importing a different version. Forgot to revert it back. I will change it now :)

Reply:
Thanks @offtherailz